index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head id=header>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Rock, Paper, Scissors</title>
+
+</head>
+
+<header>Rock x Paper x Scissors</header> 
+
+<body>
+
+<img id="rpsimage" src=https://www.esquireme.com/sites/default/files/styles/full_img/public/images/2017/05/29/rock_paper_scissors__2x.png?itok=EB8fRWP9>
+
+
+     <div id="throws-text" text-align:center>
+         <label>
+             <input id="throw" type=radio name=throw alt="rock"></intput>
+             <span>Rock</span>
+        </label>
+
+        <label>
+            <input id="throw" type=radio name=throw alt="scissors"></intput>
+            <span>Scissors</span>
+        </label>
+
+        <label>
+            <input id="throw" type=radio name=throw alt="paper"></intput>
+        <span>Paper</span>
+        </label>
+    
+    <br></br>
+
+    <button id="throw-button">Shoot!</button></div>
```

Comment:
the button tag should be indented and the closing div tag should be on the line below